advocacy_docs/edb-postgres-ai/analytics/how_to_lakehouse_sync.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Lakehouse Sync
+navTitle: Lakehouse Sync
+description: How-to guide for performing a Lakehouse Sync.
+---
+
+## Overview
+
+A Lakehouse Sync is a way to capture information from a transactional database at a point in time and sync that information to a Managed Store Location in an AWS region.
+
+Once synced, the data in the MSL is organized into Lakehouse tables. These Lakehouse tables can then be queried by a Lakehouse node optimized for higher-performance queries using a vectorized query engine for Lakehouse tables.
+
+## Perfroming a Lakehouse Sync
+
+### Prerequisites
+
+- a Postgres cluster running on your EDB Postgres AI Cloud Service account
```

Comment:
Since this is the first mention of the product, we can add the trademark symbol
```suggestion
- a Postgres cluster running on your EDB Postgres AI® Cloud Service account
```

Reply:
> Is this hosted or managed?

both right? You must have a BigAnimal instance.